packages/contentful--app-scripts/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@tsconfig/node16/tsconfig.json",
+  "exclude": ["node_modules", "**/*.test.ts"],
+  "compilerOptions": {
+    "target": "es2016",
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "outDir": "./lib",
+    "typeRoots": ["./typings", "./node_modules/@types"]
```

Comment:
Can we try to remove as many options as possible here, so our main config comes from `@tsconfig/node16`? Besides `outDir`, we can probably drop everything.

Reply:
i've just copied from create-contentful-app to be consistent, but ok, will clean this up.